frontend/portals/SearchSuggestions/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, {
+  memo, useEffect, useMemo, useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'glamor';
```

Comment:
loading spinner is hidden under TabBar
![image](https://user-images.githubusercontent.com/5811286/82347791-5a0d9480-99f8-11ea-823f-b991dd0d0020.png)